public/keymaps/s/snes_macropad_default.json

```
@@ -0,0 +1,35 @@
+{
+  "keyboard": "snes_macropad",
+  "keymap": "default",
+  "commit": "39d0a14258cbd1dd640405cdbc806dadb01521a8",
+  "layout": "LAYOUT",
+  "layers": [
+    [
+      "KC_P7",   "KC_P8",   "KC_P9",       "TO(1)"
+    , "KC_P4",   "KC_P5",   "KC_P6",       "LT(2, KC_PCMM)"
```

Configurator will treat this `LT()` key as `LT(2,KC_NO)` if there's a space here.

```suggestion
    , "KC_P4",   "KC_P5",   "KC_P6",       "LT(2,KC_PCMM)"
```